app/src/components/redirects/RootComponent.tsx

```diff
@@ -16,7 +17,17 @@ const RootComponent: React.FC = () => {
 
   const isOpenedInDesktopMode = PATHS.ROOT === location.pathname && appMode === GLOBAL_CONSTANTS.APP_MODES.DESKTOP;
 
-  return <Navigate to={isOpenedInDesktopMode ? PATHS.DESKTOP.INTERCEPT_TRAFFIC.ABSOLUTE : PATHS.HOME.ABSOLUTE} />;
+  return (
+    <Navigate
+      to={
+        isOpenedInDesktopMode
+          ? PATHS.DESKTOP.INTERCEPT_TRAFFIC.ABSOLUTE
+          : isAppTypeSessionBear()
+          ? PATHS.SESSIONS.ABSOLUTE
+          : PATHS.HOME.ABSOLUTE
+      }
+    />
+  );
```

Comment:
why not let it open in desktop as well? there is no harm and logic is also easier to read.

```suggestion
  return (
    <Navigate
      to={
        isAppTypeSessionBear() ? PATHS.SESSIONS.ABSOLUTE : 
        isOpenedInDesktopMode
          ? PATHS.DESKTOP.INTERCEPT_TRAFFIC.ABSOLUTE
          : PATHS.HOME.ABSOLUTE
      }
    />
  );
```